using_images/s2i_images/dot_net_core.adoc

```diff
@@ -94,3 +94,41 @@ ifdef::openshift-enterprise[]
 The `oc new-app` command can detect .NET Core source starting in {product-title} 3.3.
 ====
 endif::openshift-enterprise[]
+
+[[dot-net-core-templates]]
+== .NET Core Templates
+
+[IMPORTANT]
+====
+The
+link:https://github.com/redhat-developer/s2i-dotnetcore/blob/master/templates[.NET
+image templates] and the .NET images streams must first be
+link:https://github.com/redhat-developer/s2i-dotnetcore#openshift-templates[installed].
+If you ran a standard installation, the templates and image streams will be
+present. This can be checked with:
+
+----
+$ (oc get -n openshift templates; oc get -n openshift is) | grep dotnet
+----
+====
```

Comment:
This can probably be ifdef'd for just OCP/Origin. Then again this whole section in general for Dedicated/Online depends on whether it's included by default or what.